src/planet-a/Active/SysPlanetA.cpp

```diff
@@ -210,9 +245,123 @@ void SysPlanetA::update_geometry()
                                  MeshVisualizer3D::Normal{})
                 .setIndexBuffer(planet.m_indxBufGL, 0,
                                 MeshIndexType::UnsignedInt)
-                .setCount(planet.m_planet.calc_index_count());
+                .setCount(rPlanetGeo.calc_index_count());
+        }
+
+        if (m_debugUpdate.triggered() || true)
```

Comment:
So... always?

Reply:
Yep. It was originally configured in a way that the planet will update only when pressing a button. As it became more complete, the `|| true` was added so it updates every frame.

Reply:
well, i'd recommend removing the if-check then